### PR TITLE
Feature/final touches

### PR DESCRIPTION
**Pull request by ubong-s**

### CODE REVIEW

File: src/pages/blog/[...slug].astro [hunk 1]
```diff
@@ -20,7 +20,8 @@ const { title, description, image } = post.data;
   title={title}
   meta_title={title}
   description={description}
-  image={image ?? placeholder}
+  image={image && image.src}
```
Comment (shippy) [inline src/pages/blog/[...slug].astro:23]: Should the placeholder still apply if `image` is falsey?

File: src/pages/presentations/[...slug].astro [hunk 2]
```diff
@@ -20,7 +20,7 @@ const { title, description, heroImage } = presentation.data;
   title={title}
   meta_title={title}
   description={description}
-  image={heroImage ?? placeholder}
+  image={heroImage && heroImage.src}
```
Comment (shippy) [inline src/pages/presentations/[...slug].astro:23]: Same issue - if the `heroImage` is absent, shouldn't there be a `placeholder` backup?

**Comment by ubong-s** [comment]
There is a placeholder available site wide so those weren’t necessary
anymore. Check the “Base” Layout. The fallback is there

**Comment by ubong-s** [comment]
https://github.com/ubong-s/shippy.github.io/blob/feature/final_touches/src/layouts/Base.astro

**Comment by ubong-s** [comment]
Fallback 2
Fallback

File: src/pages/blog/[...slug].astro (hunk 1 above)
Comment (ubong-s) [reply src/pages/blog/[...slug].astro:23]: I can add it back if you want

**Comment by shippy** [comment]
I see - thanks for explaining!

**Review by shippy — APPROVED** [review]

**Pull request merged**